specs/MultiProfile.md

```diff
@@ -247,6 +320,13 @@ interface ICoreWebView2Profile : IUnknown {
 
   // TODO: All profile-wide operations/settings will be put below in the future.
 }
+
+[uuid(B93875C2-D6B0-434D-A2BE-93BC06CCC469), object, pointer_default(unique)]
+interface ICoreWebView2Profile2 : ICoreWebView2Profile {
+  /// Get the cookie manager object for the profile. All CoreWebView2s associated with this profile share this same cookie manager and will have the same CoreWebView2.CookieManager property value.
```

Comment:
Still confused; there is no CoreWebView2.CookieManager property, right?

Reply:
I mean, each profile is associated with one or more WebView2s (created using this profile). All the WebView2s (which are associated with this profile) share the same cookie manager.
I made a mistake, Optionally we can get webView.CoreWebView2.CookieManager in .NET and WinRT, maybe I should write "will have the same ICoreWebView2CookieManager object" here?

Reply:
There is a [CoreWebView2.CookieManager property](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.cookiemanager?view=webview2-dotnet-1.0.1056-prerelease). It was added before we had a profile class. This new CoreWebView2Profile.CookieManager class is as noted above the same value for all the CoreWebView2s in the same profile. CoreWebView2.CookieManager will be the same as CoreWebView2.Profile.CookieManager.

Reply:
So is it fine to keep it here? It is in **Win32 C++ API Details** section and we mention the property in the .NET form.

Reply:
Got it, thanks